shared/data/etags.json

```diff
@@ -1 +1 @@
-{"config-etag":"W/\"260cf6ba6d4ac2cf2e2a5ef94c9edcdc\""}
\ No newline at end of file
+{"config-etag":"W/\"f1fa2ea9d9d27db1c7a68d7a81f6242d\""}
```

Comment:
Why has the etag changed when the content has not? Can we just drop these data changes from this diff?

Reply:
Me messing about, removed.